doc/man3/RSA_padding_add_PKCS1_type_1.pod

```diff
@@ -105,6 +105,12 @@ The RSA_padding_check_xxx() functions return the length of the
 recovered data, -1 on error. Error codes can be obtained by calling
 L<ERR_get_error(3)>.
 
+=head1 WARNING
+
+The RSA_padding_check_PKCS1_type_2() padding check leaks timing
+information which can potentially be used to mount a Bleichenbacher
+padding oracle attack. Prefer PKCS1_OAEP padding.
```

Comment:
It's certainly a flaw in the PKCS1 structure, but the pertinent problem here is that the API is unsafe. One could produce alternative APIs which are tied to a specific mitigation or that are designed to be used in concert with the caller's constant-time code which implements a mitigation, but that is not this API.